.travis.yml

```diff
@@ -43,6 +44,7 @@ before_script:
 script:
   - export PATH=$HOME/.cargo/bin:$PATH
   - make ci-travis
+  - make emulation-check
```

Comment:
Is there a reason to put this here rather than inside of the Makefile? I don't like that this now means `make ci-travis` != run locally what runs on travis.

Given the aspiration of multi-platform travis, the make rule might need to be something like:

```diff
.PHONY: ci-travis
ci-travis:\
	ci-formatting\
	ci-tools\
	ci-libraries\
	ci-archs\
	ci-kernel\
	ci-chips\
	ci-syntax\
	ci-compilation\
	ci-debug-support-targets\
	ci-documentation
+        @if [[ $TRAVIS_OS_NAME == "linux" ]]; $(MAKE) emulation-check; else; printf "$$(tput bold)Skipping QEMU, unsupported platform(tput sgr0)\n"; fi
	@printf "$$(tput bold)********************$$(tput sgr0)\n"
	@printf "$$(tput bold)* CI-Travis: Done! *$$(tput sgr0)\n"
	@printf "$$(tput bold)********************$$(tput sgr0)\n"
```

Reply:
The reason was that maybe QEMU builds are too slow for some people and if they are just making ARM related changes they don't need to run RISC-V tests locally.

Are we targeting a platform where QEMU won't build?

Reply:
I think I would rather preserve the idiom that `make ci-travis` is exactly what travis runs.

Putting QEMU last hopefully mitigates any build time concerns -- in the hopefully-not-too-distant github actions future, that won't be an issue anyway.

---

We have a goal of running travis on OS X on master-only https://github.com/tock/tock/pull/1625; though again, maybe with github actions, that looks different as well. 

> OH, QEMU works on Mac, interesting

Umm.. I had it in my head that it didn't work on OS X; let me try getting this running locally once

Reply:
Ok, I have moved it to the ci Make target.

QEMU should build and run on OS X. I don't think we would need any changes to the scripts to have it work.